site/content/concepts/resource-limits.md

```diff
@@ -0,0 +1,18 @@
++++
+title = "Resource limits"
+weight = 196
++++
+
+Eclipse Hono™ users can set the resource limits such as the maximum number of device connections allowed per tenant or the allowed data volume of the messages over a period of time per tenant. 
```

Comment:
we usually do not use this style of *active* writing (`Hono user can set ...`) in the documentation. Instead, we usually use a *passive* style, e.g. 
> Resource limits can be set using ...

Can you please rephrase here and in the other pages in order to maintain a consistent style?